index.js

```diff
@@ -3,10 +3,49 @@ const cors = require("cors");
 const knex = require("knex");
 const knexConfig = require("./knexfile.js");
 const bcrypt = require("bcryptjs");
+const session = require("express-session");
+const KnexSessionStore = require("connect-session-knex")(session);
 const db = knex(knexConfig.development);
 const server = express();
 server.use(express.json());
 server.use(cors());
+const sessionConfig = {
+  name: "login",
+  secret: "I eat whales",
+  cookie: {
+    loggedIn: true,
```

Comment:
What does this `loggedIn` property do in the `cookie` object? Can't find anything in the documentation.